examples/byovnet/README.md

```diff
@@ -0,0 +1,200 @@
+<!-- BEGIN_TF_DOCS -->
+# BYO Vnet example
+
```

Comment:
May be we should add an example of using an existing VNET?

Reply:
We could add that as a ticket to the board?